Makefile.toml

```diff
@@ -713,7 +713,7 @@ ${TMUX} list-windows -F "#{window_name} #{pane_id}" \
 | xargs -I {} ${TMUX} send-keys -t {} C-c
 
 # Stop docker components
-containers=$(docker ps -a -q -f name=risedev- 2>/dev/null) || true
+containers=$(docker ps -q -f name='risedev|risingwave'- 2>/dev/null) || true
```

Comment:
The `risedev k` command is only supposed to kill services started by `risedev` 🤔. But I don't mind too much it killing other risingwaves anyway.

Reply:
I find it a bit weird though... since running two instances does not imply they are working on the same ports.

What if the developer intends to run one production instance with `docker` and one development instance with `risedev` together, for example, to compare their behaviors?

Reply:
I guess a better way is to show the name of process which uses the port we need, and let the user to decide what to do next.

Reply:
> What if

I don't like to talk about imaginary problems, and I don't have preferences on this specific issue. 😄

> I guess a better way is to show the name of process which uses the port we need, and let the user to decide what to do next.

elegant!